index.src.html

```diff
@@ -916,21 +930,25 @@ <h4 algorithm id="collectfromcredentialstore-passwordcredential">
   </ol>
 
   <h4 algorithm id="create-passwordcredential">
-    `PasswordCredential`'s `[[Create]](data)`
+    `PasswordCredential`'s `[[Create]](options)`
   </h4>
 
-  <dfn for="PasswordCredential" method>\[[Create]](data)</dfn> is called with a
-  {{CredentialCreationData}} (|data|), and returns a {{PasswordCredential}} if one can be created,
-  or `null` otherwise:
+  <dfn for="PasswordCredential" method>\[[Create]](options)</dfn> is called with a
+  {{CredentialCreationOptions}} (|options|), and returns a {{PasswordCredential}} if one can be created,
+  or `null` otherwise. The {{CredentialCreationOptions}} dictionary must have a `password` member which
```

Comment:
Should we mention that exceptions can be thrown? (see line 1079 in the new version)